pca/db/database_model.py

```diff
@@ -84,7 +86,7 @@ class Meta:
         collection_name = CUSTOMER_COLLECTION
         final = True  # so we don't get a '_cls' field in these documents (e.g. "_cls" : "__main__.Customer")
 
-    def get_all_customers():
+    def get_all_customers(self):
```

Comment:
Is this change necessary?  Can this method remain static?

Reply:
Yes, that can remain static. I'll make the change.